### PR TITLE
Add Python MongoDB pipelines for read/write

### PR DESCRIPTION
**Pull request by Amar3tto**
Pipelines added:

1. **Read from MongoDB**
IO: MongoDB IO
SDK Language: Python
Run instruction:
- Start MongoDB
- Create collection and fill with test data
- run `python read_mongodb.py --runner="DataflowRunner" --project="your-project" --region="your-region" --uri="your-uri" --db_name="your-db-name" --collection="your-collection"`

2. **Write to MongoDB**
IO: MongoDB IO
SDK Language: Python
Run instruction:
- Start MongoDB
- Create collection
- run `python write_mongodb.py --runner="DataflowRunner" --project="your-project" --region="your-region" --uri="your-uri" --db_name="your-db-name" --collection="your-collection"`

### CODE REVIEW

**Review by olehborysevych — APPROVED** [review]
LGTM
Please update PR description and move from draft

**Review by Dippatel98 — APPROVED** [review]
LGTM

File: Python/mongodb/read_mongodb.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    @classmethod
+    def _add_argparse_args(cls, parser):
+        # Add a command line flag to be parsed along
+        # with other normal PipelineOptions
+        parser.add_argument(
+            "--uri",
+            default="mongodb://localhost:27017",
+            help="The MongoDB connection string following the URI format"
+        )
+        parser.add_argument(
```
Comment (liferoad) [inline Python/mongodb/read_mongodb.py:35]: both db name and coll name should be required without default, right?

**Comment by liferoad** [comment]
Please resolve the conflict.

File: Python/mongodb/read_mongodb.py (hunk 1 above)
Comment (Amar3tto) [reply Python/mongodb/read_mongodb.py:35]: Fixed

**Review by liferoad — APPROVED** [review]

**Pull request merged**